packages/amplication-e2e-tester/Login test/login.test.js

```diff
@@ -0,0 +1,24 @@
+const TITLE = "Amplication";

```

Comment:
I don't think we currently need test files to be in dedicated directories. Please move the tests to a single "tests" directory